src/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -68,6 +68,7 @@ public static class MsalError
         /// This error code comes back from <see cref="ClientApplicationBase.AcquireTokenSilent(System.Collections.Generic.IEnumerable{string}, IAccount)"/> calls when
         /// the user cache had not been set in the application constructor. This should never happen in MSAL.NET 3.x as the cache is created by the application
```

Comment:
This comment shows that this is an ADAL concept that was "pushed" into MSAL. We just never cleaned it up after the divorce.